### ISSUE TITLE
SafePrimes keyver failed to generate vectors

### ISSUE THREAD

**Issue by smuellerDD** [usnistgov/ACVP#803]
**environment**
demo

**testSessionId**
75390

**vsId**
160033

**Algorithm registration**
```
      {
        "revision":"1.0",
        "algorithm":"safePrimes",
        "mode":"keyVer",
        "prereqVals":[
          {
            "algorithm":"DRBG",
            "valValue":"same"
          }
        ],
        "safePrimeGroups":[
          "MODP-2048",
          "MODP-3072",
          "MODP-4096",
          "MODP-6144",
          "MODP-8192"
        ]
      }
```

**Endpoint in which the error is experienced**
GETing the test vectors

**Expected behavior**
vector returned

**Additional context**
Error:

```
[ {
  "acvVersion" : "1.0"
}, {
  "vsId" : 160033,
  "status" : "error during vector generation",
  "message" : "failed to generate tests for: 160033"
} ]
```

**Comment by Kritner** [comment]
Hey @smuellerDD I see an issue on our end regarding the safe prime groups `> 2048`, I'll put in a fix in the next release - also relates to #801.  It looks like you should be ok to test with the groups that were `<= 2048` until that fix is in.

Tagging #685 for visibility

**Comment by Kritner** [comment]
A push related to this has been pushed out to demo.  Let me know if there are any further issues.
